### PR TITLE
Dev Screen Fixes + Accordion fixes

### PR DESCRIPTION
**Pull request by elfenheart**
Add buttons to Dev Screen:
- Add ETL Apps
- Add Stream

Show Total number of Data and Apps

screen shot 2015-04-15 at 11 38 19 pm
screen shot 2015-04-15 at 11 38 56 pm

Defaulting First Accordion to open
accordions

### CODE REVIEW

File: cdap-ui/app/features/workflows/controllers/tabs/schedule-ctrl.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ angular.module(PKG.name + '.feature.workflows')
         });
       });
 
+      $scope.schedules[0].isOpen = true;
```
Comment (anwar6953) [inline cdap-ui/app/features/workflows/controllers/tabs/schedule-ctrl.js]: If there are no schedules?

Comment (elfenheart) [inline cdap-ui/app/features/workflows/controllers/tabs/schedule-ctrl.js]: Addressed

File: cdap-ui/app/features/overview/templates/data-apps.html [hunk 2]
```diff
@@ -4,7 +4,8 @@
       <h3 class="text-uppercase"> Data </h3>
       <div class="row">
         <div class="col-xs-12 col-sm-5">
-          <p> Most recent </p>
+          <p ng-if="streams.concat(datasets).length < 5"> Showing {{ streams.concat(datasets).length }} out of {{ streams.concat(datasets).length }}</p>
```
Comment (anwar6953) [inline cdap-ui/app/features/overview/templates/data-apps.html]: we're doing streams.concat(datasets) 5 times in these 2 lines.
Does it make sense to do the concatenation once in the controller and use that object in the 5 places here?

Comment (nitinmotgi) [inline cdap-ui/app/features/overview/templates/data-apps.html]: +1

**Comment by ajainarayanan** [comment]
:+1: LGTM apart from @anwar6953's comments.

**Comment by elfenheart** [comment]
Comments addressed

**Pull request merged**